python/cugraph-equivariant/cugraph_equivariant/nn/tensor_product_conv.py

```diff
@@ -74,26 +79,28 @@ class FullyConnectedTensorProductConv(nn.Module):
     Examples
     --------
     >>> # Case 1: MLP with the input layer having 6 channels and 2 hidden layers
-    >>> #         having 16 channels. edge_emb.size(1) must match the size of
-    >>> #         the input layer: 6
+    >>> # having 16 channels. edge_emb.size(1) must match the size of the input layer: 6
     >>>
     >>> conv1 = FullyConnectedTensorProductConv(in_irreps, sh_irreps, out_irreps,
     >>>     mlp_channels=[6, 16, 16], mlp_activation=nn.ReLU()).cuda()
     >>> out = conv1(src_features, edge_sh, edge_emb, graph)
     >>>
     >>> # Case 2: Same as case 1 but with the scalar features from edges, sources
-    >>> #         and destinations passed in separately.
+    >>> # and destinations passed in separately. This option allows a smaller GEMM
+    >>> # in the first MLP layer by performing GEMM on each component before indexing.
+    >>> # The first-layer weights are split into sections for edges, sources and
+    >>> # destinations, in that order.
```

Comment:
(in that order), which are to be multiplied with the respective input of `edge_emb`, `src_scalars` and `dst_scalars`. The following example is equivalent to calling the forward function in usage case 1 with  `conv1(src_features, edge_sh, torch.hstack([edge_emb, src_scalars[graph[0][0]], dst_scalars[graph[0][1]]]) , graph)`.